scaper/util.py

```diff
@@ -11,6 +11,8 @@
 from .scaper_exceptions import ScaperError
 import scipy
 import numpy as np
+import numbers
+import random
```

Comment:
`import random` shouldn't be here, right?

Reply:
Whoops! Fixed.